core/src/main/scala/org/broadinstitute/dsde/rawls/snapshot/SnapshotService.scala

```diff
@@ -36,7 +36,7 @@ class SnapshotService(protected val userInfo: UserInfo, val dataSource: SlickDat
         workspaceManagerDAO.createWorkspace(workspaceContext.workspaceId, getServiceAccountAccessToken, userInfo.accessToken)
       }
 
-      val dataRepoReference = JsObject.apply(("instance", JsString(terraDataRepoUrl)), ("snapshot", JsString(snapshot.snapshotId)))
+      val dataRepoReference = JsObject("instance" -> JsString(terraDataRepoUrl), "snapshot" -> JsString(snapshot.snapshotId)).toString()
```

Comment:
could/should this be a case class with `instance` and `snapshot` members, instead of the raw `JsObject`?